### DISCUSSION TITLE
Now that postgres has PROCEDUREs as well as FUNCTIONs ... can Scenic?

### DISCUSSION THREAD

**Discussion by redaikidoka** [scenic-views/scenic#392] [Ideas]
It would be amazing if scenic supported them!

I would love to be able to say

`rails g fx:procedure my_fancy_procedure`

Because I accidentally tried to do that with a procedure and use `create_function` / `update_function` and boy did that not work (because of course it isn't supported yet).

I love this gem!

**Comment by derekprior** [comment]
Hi, check out https://github.com/teoljungberg/fx which has similar underpinnings
